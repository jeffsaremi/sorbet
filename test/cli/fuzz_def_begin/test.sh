#!/usr/bin/env bash

set -euo pipefail

if main/sorbet --silence-dev-message 2>&1 test/cli/fuzz_def_begin/test.rb; then
  echo "Expected to fail"
  exit 1
fi
