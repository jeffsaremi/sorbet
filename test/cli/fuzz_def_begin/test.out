test/cli/fuzz_def_begin/test.rb:2: unexpected token "end of file" https://srb.help/2001
     2 |def BEGIN
     3 |

test/cli/fuzz_def_begin/test.rb:2: Hint: this "def" token is not closed before the end of the file https://srb.help/2003
     2 |def BEGIN
        ^^^
Errors: 2
